<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="box_wav">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="title">
        <string>Wav</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="box_uart">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Uart</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
        <item>
         <widget class="QPlainTextEdit" name="textbrowser_uart_receive">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineedit_uart_send"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="uartProfileBox">
      <property name="title">
       <string>Profile</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="box_uart_setting">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Uart Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>串口号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>波特率</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>数据位</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>停止位</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QComboBox" name="combo_uart_setting_serail_no"/>
              </item>
              <item>
               <widget class="QComboBox" name="combo_uart_setting_burdrate"/>
              </item>
              <item>
               <widget class="QComboBox" name="combo_uart_setting_databit"/>
              </item>
              <item>
               <widget class="QComboBox" name="combo_uart_setting_stopbit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkbox_uart_setting_dtr">
              <property name="text">
               <string>DTR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkbox_uart_setting_rts">
              <property name="text">
               <string>RTS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_uart_setting_more">
              <property name="text">
               <string>更多</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_uart_setting_open">
            <property name="text">
             <string>打开串口</string>
            </property>
            <property name="icon">
             <iconset resource="wadcs.qrc">
              <normaloff>:/images/uart-state-off.png</normaloff>:/images/uart-state-off.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="box_uart_rcv">
         <property name="title">
          <string>Uart Receive</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QCheckBox" name="checkbox_uart_rcv_hex_display">
              <property name="text">
               <string>Hex显示</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkbox_uart_rcv_save_to_file">
              <property name="text">
               <string>存入文件</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkbox_uart_rcv_add_timestamp">
            <property name="text">
             <string>加时间戳和分包显示</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QToolButton" name="btn_uart_rcv_stop">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>停止</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_uart_rcv_clear">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>清除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="box_uart_send">
         <property name="title">
          <string>Uart Send</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QCheckBox" name="checkbox_uart_send_hex">
              <property name="text">
               <string>Hex发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_uart_send_choose_file">
              <property name="text">
               <string>选择文件</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkbox_uart_send_timely">
              <property name="text">
               <string>定时发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkbox_uart_send_newline">
              <property name="text">
               <string>发送新行</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="edit_uart_send_timely">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>ms/次</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_uart_send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="wadcs.qrc"/>
 </resources>
 <connections/>
</ui>
